lib/apple/index.js

```diff
@@ -112,8 +116,8 @@ exports.verifyPayment = function (payment, cb) {
 		return cb(null, result);
 	}
 
-
-	verify(apiUrls.production, { json: jsonData }, function (error, resultString) {
+	var url = sandbox ? apiUrls.sandbox : apiUrls.production;
```

Comment:
@ronkorving are we still looking good for getting this merged? If we simply remove the url assignment and rely on what you suggested, would that be enough to get a `0.5.1` release?